.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,59 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    name: Release Please
+    runs-on: ubuntu-latest
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
I left this on v3 because v4 still has a few issues.

Reply:
Can you point me toward what the issues with v4 are? Do we know if the maintainers have it prioritized/are working on it?

Reply:
There are a few minor issues that we could likely address. The first relates to the releases_created [output](https://github.com/google-github-actions/release-please-action/issues/912). You also can't have the bot sign off on a comment. This fix will come once [this release](https://github.com/google-github-actions/release-please-action/pull/969) is merged.

I'm not sure if there are other issues. I moved away from v4 once I discovered the limited listed above.